### PR TITLE
API documentation: some fixes

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Intro fixes to some of the documentation in the API area. I wanted to simplify the main API landing page as well as remove the API key setup docs, which are not needed at this time. Feedback welcome!

### CODE REVIEW

File: api/index.md [hunk 1]
```diff
@@ -9,27 +9,24 @@ has_children: true
...
-The **Data Commons API** is a set of APIs that allow developers to
+The **Data Commons API** allows developers to
 programmatically access the data in Data Commons.
 Data Commons provides APIs in each of the different views
 (graph, timeline, etc.) for use in different contexts. More specifically, we provide
-1. **REST** APIs that can be used by javascript based applications
-1. **Pandas** APIs for use in Python notebooks
-1. **Google Sheets** APIs for use in Google Sheets
-1. **Python** APIs
-1. **SQL** (Coming soon)
```
Comment (KilimAnnejaro) [inline api/index.md:22]: Per https://developers.google.com/style/future, removing future feature documentation. (If needed, I can write up a blog post for roadmap if that would be helpful.)

File: api/setup.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ layout: default
 title: Setting up the API
 parent: API
 nav_order: 1
+published: false
```
Comment (KilimAnnejaro) [inline api/setup.md:6]: Most of this documentation relates to API key setup, which is not needed at this time.

File: api/index.md [hunk 3]
```diff
@@ -9,105 +9,30 @@ has_children: true
...
-The **Data Commons API** is a set of APIs that allow developers to
-programmatically access the data in Data Commons.
-Data Commons provides APIs in each of the different views
-(graph, timeline, etc.) for use in different contexts. More specifically, we provide
-1. **REST** APIs that can be used by javascript based applications
-1. **Pandas** APIs for use in Python notebooks
-1. **Google Sheets** APIs for use in Google Sheets
-1. **Python** APIs
-1. **SQL** (Coming soon)
+The **Data Commons API** allows developers to programmatically access the data in Data Commons.
```
Comment (tjann) [inline api/index.md:14]: This is very well written. I just wonder if it's helpful to users to see the different endpoints highlighted (maybe at least bold/italics if we want to remove the bullet points)?

Comment (tjann) [reply api/index.md:14]: Maybe even with links to the docsite pages for each endpoint

**Review by tjann — APPROVED** [review]

File: api/index.md (hunk 3 above)
Comment (KilimAnnejaro) [reply api/index.md:14]: I've changed the docs per this suggestion. Let me know what feels best to you!

Comment (tjann) [reply api/index.md:14]: looks good!

**Pull request merged**